src/lexer.cpp

```diff
@@ -28,8 +28,12 @@ void Lexer::push_token(token_type tt, int width) {
 	for (int i = 0; i < width; ++i)
 		text[i] = m_source[m_source_index + i];
 
-	Token t = { tt, std::move(text), m_current_line, m_current_column, m_current_line,
-		m_current_column + width };
+	Token t = { tt,
+		        std::move(text),
+		        m_current_line,
+		        m_current_column,
+		        m_current_line,
+		        m_current_column + width };
```

Comment:
This I completely despise

Reply:
I am pretty sure this is a bug in `clang-format`

Reply:
`Cpp11BracedListStyle: true` should make braced lists format the same than parenthesized lists, but this is not the case.

No matter how much I fiddle with configuration, I still get this style:

```c++

auto t = Token(
	tt,
	std::move(text),
	m_current_line,
	m_current_column,
	m_current_line,
	m_current_column + width);

auto t = Token { tt,
                 std::move(text),
                 m_current_line,
                 m_current_column,
                 m_current_line,
                 m_current_column + width };
```

I want the second declaration to be formatted the same than the first one. but it keeps getting aligned to the right instead of to the left.

Reply:
Yeah. This was caused by a bug in clang-format-9. I upgraded my installation to clang-format-10, and it's good now.